src/main/java/edu/greenblitz/robotName/OI.java

```diff
@@ -21,6 +23,14 @@ public static OI getInstance() {
         return instance;
     }
 
+    public void initButt(){
+        butt();
+    }
+
+    public void butt(){
+        secondJoystick.B.whileTrue(new runMotors());
+    }
+
```

Comment:
merge to 1 functuion
```suggestion
    public void initPrototypeButtons(){
        secondJoystick.B.whileTrue(new RunMotors);
    }

```